test/e2e/allocation.new.test.js

```diff
@@ -0,0 +1,25 @@
+import { pmuUser } from './_roles'
+import { newAllocation } from './_routes'
+import { allocationJourney } from './pages/'
+
+fixture('New PMU allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+})
+test('Create allocation with all the options', t =>
```

Comment:
I think it would be good to actually test the result of the creation as part of this test.

Maybe the `createAllocation` method should just create the allocation rather than be concerned with assertions, the current name is a bit misleading.

If you have a look at the creating of moves, the test files call page object methods to do particular interactions and end by calling methods that assert particular behaviour.

Reply:
I can't quite see how it's misleading... it's what it does! However, it's a good idea to test its result, and I have now added it.